content/applications/inventory_and_mrp/inventory/barcode/operations/transfers_scratch.rst

```diff
@@ -1,17 +1,200 @@
+
+Once the desired locations have been selected, products can be added to the transfer. On the
+:guilabel:`Product` line under the :guilabel:`Products` tab, click :guilabel:`Add a product`, and
+select the desired product(s) to add to the transfer.
+
+Once ready, click :guilabel:`Save` at the top of the form to save the new internal transfer. Once
+saved, click the :guilabel:`Detailed Operations` icon (four lines, at the far right of the
+:guilabel:`Product` line) to open the :guilabel:`Detailed Operations` pop-up window.
+
+.. image:: transfers_scratch/transfers-scratch-detailed-operations-popup.png
+   :align: center
+   :alt: Internal transfer detailed operations pop-up window.
+
+From the pop-up, click :guilabel:`Add a line`. Then, in the :guilabel:`To` column, change the
+location from :guilabel:`WH/Stock` to a different location, where the products should be moved.
+Then, in the :guilabel:`Done` column, change the quantity to the desired quantity to transfer. Once
+ready, click :guilabel:`Confirm` to close out the pop-up window.
+
+Scan barcodes for internal transfer
+-----------------------------------
+
+To process and scan barcodes for internal transfers, navigate to the :menuselection:`Barcode app`.
+
+Once inside the :guilabel:`Barcode app`, a :guilabel:`Barcode Scanning` screen displaying different
+options is presented. To process internal transfers, click on the :guilabel:`Operations` button at
+the bottom of the screen. This navigates to an :menuselection:`Operations` overview page.
+
+.. image:: transfers_scratch/transfers-scratch-barcode-app.png
+   :align: center
+   :alt: Barcode app start screen with scanner.
+
+From this page, locate the :guilabel:`Internal Transfers` card, and click the :guilabel:`# To
+Process` button to view all outstanding internal transfers. Then, select the desired operation to
+process. This navigates to the barcode transfer screen.
+
+.. note::
+   If *only* using a barcode scanner or the Odoo mobile app, the barcodes for each transfer of a
+   corresponding operation type can be scanned to be processed easily. Once scanned, the products
+   that are part of an existing transfer can be scanned, and new products can be added to the
+   transfer, as well. Once all products have been scanned, validate the transfer to proceed with the
+   stock moves.
+
+From this screen, an overview of all products to process within that specific internal transfer
+(**WH/INT/000XX**) is shown. At the bottom of the screen, there are options to :guilabel:`Add
+Product` or :guilabel:`Validate`, depending on if products need to be added to the operation, or if
+the whole operation should be validated at once.
+
+.. image:: transfers_scratch/transfers-scratch-receipts-overview.png
+   :align: center
+   :alt: Overview of receipts in transfer to scan.
+
+To process and scan each product individually, choose a specific product line. The :guilabel:`+#`
```

Comment:
From this screen, we can either scan the barcode or use the buttons on the screen, like u detailed (:
```suggestion
Then, scan the barcode of the product to process the internal transfer.

Or, to process and scan each product individually, choose a specific product line. The :guilabel:`+#`
```